packages/theme-light-extension/src/index.ts

```diff
@@ -1,19 +1,31 @@
+
+
 import {
-  JupyterLabPlugin
+  JupyterLab, JupyterLabPlugin
 } from '@jupyterlab/application';
 
-import '@jupyterlab/theming/style/variables-light.css';
+import {
+  IThemeManager
+} from '@jupyterlab/apputils';
 
 
-/**
- * Initialization data for the light theme extension.
- */
-const extension: JupyterLabPlugin<void> = {
-  id: 'jupyter.themes.light',
-  autoStart: true,
-  activate: (app) => {
-    // No-op.
-  }
+
+const plugin: JupyterLabPlugin<void> = {
+  id: 'theme-light-extension',
```

Comment:
What is the rationale for not calling these `jupyter.themes.light` or 'jupyter.extensions.light-theme' or some variety of `jupyter.foo.bar`?